SymSettings.cfg

```diff
@@ -1,68 +1,80 @@
 ### MAIN ###
```

Comment:
You should remove SymSettings.cfg from the PR so that we don't have it keep changing